code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -639,12 +639,13 @@
 		"Basic" = "starman",
 	)
 	speed_modifier = CYBORG_STARMAN_SPEED_MODIFIER
+	default_modules = 0
```

Comment:
No, really, at least change it to FALSE

Reply:
var/default_modules = TRUE //Do we start with a flash/light?

I could make it FALSE for clarity but I would hope a programmer would know that 0 is false...

Reply:
I changed every borg binary check to use TRUE/FALSE. Keep it consistent.

Reply:
Alright BY*ND coder